lua/utils.lua

```diff
@@ -15,13 +15,12 @@ function M.executable(name)
   return false
 end
 
-function M.may_create_dir()
-  local fpath = fn.expand('<afile>')
-  local parent_dir = fn.fnamemodify(fpath, ":p:h")
-  local res = fn.isdirectory(parent_dir)
+--- Create a dir if if does not exist
```

Comment:
typo: repeat if.

Reply:
Thanks, you have sharp eyes :). Will fix it ASAP.

Reply:
fixed in https://github.com/jdhao/nvim-config/commit/b279c0cd95bc1246726deecf1199ea3c132ea0c6